### ISSUE TITLE
[Bug] Promoted tweets in saved search after updating to 1.8, still works in 1.7

### ISSUE THREAD

**Issue by defkev** [Dr-TSNG/TwiFucker#82]
### Steps to reproduce/复现步骤

1. Install 1.8
2. Enable Disable Promoted Content
3. Open a saved search

### Expected behaviour/预期行为

Hide Promoted tweets

### Actual behaviour/实际行为

Promoted tweets in Top/Latest

### Xposed Module List/Xposed 模块列表

```Shell
MinMinGuard
TwiFucker
```

### TwiFucker version/TwiFucker 版本

v1.8

### Twitter version/Twitter 版本

9.68.1-release.0

### Android version/Android 版本

12

### Xposed Framework/Xposed 框架

LSPosed

### Framework version/框架版本

1.8.5 (6649)

### Version requirement/版本要求

Public release/beta version/公开发布/测试版

### Logs/日志

```
12-21 02:57:26.571 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@b6df865
12-21 02:57:26.574 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 1 line
12-21 02:57:26.574 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@b6df865
12-21 02:57:26.578 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@23a6abf
12-21 02:57:26.580 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 1 line
12-21 02:57:26.580 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@23a6abf
12-21 02:57:26.583 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@dda758e
12-21 02:57:26.585 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 1 line
12-21 02:57:26.585 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@dda758e
12-21 02:57:29.730 11352 11516 D ProfileInstaller: Skipping profile installation for com.twitter.android
:/storage/emulated/0/Android/data/com.twitter.android/cache/twifucker_log # tail -f log.txt
12-21 02:57:26.571 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@b6df865
12-21 02:57:26.574 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 1 line
12-21 02:57:26.574 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@b6df865
12-21 02:57:26.578 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@23a6abf
12-21 02:57:26.580 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 1 line
12-21 02:57:26.580 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@23a6abf
12-21 02:57:26.583 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@dda758e
12-21 02:57:26.585 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 1 line
12-21 02:57:26.585 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@dda758e
12-21 02:57:29.730 11352 11516 D ProfileInstaller: Skipping profile installation for com.twitter.android
12-21 02:57:57.649 11352 11352 I PlayCore: UID: [10177]  PID: [11352] SplitInstallListenerRegistry : unregisterListener
12-21 02:57:57.687 11352 11352 W TabLayout: MODE_SCROLLABLE + GRAVITY_FILL is not supported, GRAVITY_START will be used instead
12-21 02:57:57.724 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@b4f172c
12-21 02:57:57.726 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@b6fd9fb
12-21 02:57:57.729 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@bc9a56
12-21 02:57:57.746 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@5aa564a
12-21 02:57:57.765 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@7237788
12-21 02:57:57.781 11352 11352 W View    : requestLayout() improperly called by com.google.android.material.tabs.TabLayout$f{e0f4966 V.ED..... ......ID 0,0-1080,126} during layout: running second layout pass
12-21 02:57:57.781 11352 11352 W View    : requestLayout() improperly called by com.google.android.material.tabs.TabLayout$i{fc384a7 VFE...C.. ..S..AID 0,0-211,126} during layout: running second layout pass
12-21 02:57:57.781 11352 11352 W View    : requestLayout() improperly called by com.google.android.material.tabs.TabLayout$i{8d48d54 VFE...C.. ......ID 211,0-422,126} during layout: running second layout pass
12-21 02:57:57.781 11352 11352 W View    : requestLayout() improperly called by com.google.android.material.tabs.TabLayout$i{887f4fd VFE...C.. ......ID 422,0-633,126} during layout: running second layout pass
12-21 02:57:57.781 11352 11352 W View    : requestLayout() improperly called by com.google.android.material.tabs.TabLayout$i{73695f2 VFE...C.. ......ID 633,0-844,126} during layout: running second layout pass
12-21 02:57:57.781 11352 11352 W View    : requestLayout() improperly called by com.google.android.material.tabs.TabLayout$i{f19c443 VFE...C.. ......ID 844,0-1055,126} during layout: running second layout pass
12-21 02:57:57.850 11352 11352 W IInputConnectionWrapper: requestCursorAnchorInfo on inactive InputConnection
12-21 02:57:59.105 11352 11352 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
12-21 02:57:59.107 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:57:59.115 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@442c2e3
12-21 02:57:59.120 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:57:59.120 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:57:59.122 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:57:59.122 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:57:59.123 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:57:59.123 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:57:59.123 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:57:59.124 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 29 lines
12-21 02:57:59.124 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:57:59.125 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@b5dd055
12-21 02:57:59.125 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:57:59.125 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:57:59.148 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@442c2e3
12-21 02:57:59.148 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 6 lines
12-21 02:57:59.148 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@442c2e3
12-21 02:57:59.171 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@7445a53
12-21 02:57:59.224 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@442c2e3
12-21 02:57:59.224 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 1 line
12-21 02:57:59.224 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@442c2e3
12-21 02:57:59.225 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@7445a53
12-21 02:57:59.225 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@7445a53
12-21 02:57:59.225 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@4ad2feb
12-21 02:57:59.226 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 4 lines
12-21 02:57:59.226 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@4ad2feb
12-21 02:57:59.228 11352 11352 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
12-21 02:57:59.231 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:57:59.236 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@fa4ff8d
12-21 02:57:59.238 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:57:59.238 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:57:59.239 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:57:59.239 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:57:59.240 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:57:59.240 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:57:59.241 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:57:59.241 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 29 lines
12-21 02:57:59.241 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:57:59.242 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@17db953
12-21 02:57:59.242 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:57:59.242 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:57:59.249 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@fa4ff8d
12-21 02:57:59.278 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 9 lines
12-21 02:57:59.278 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@fa4ff8d
12-21 02:58:02.212 11352 11352 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
12-21 02:58:02.214 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:02.220 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@ce88ad9
12-21 02:58:02.223 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:02.223 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:02.224 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:02.224 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:02.225 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:02.225 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:02.225 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:02.226 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 29 lines
12-21 02:58:02.226 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:02.227 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@215937f
12-21 02:58:02.227 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:02.227 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:02.234 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@ce88ad9
12-21 02:58:02.259 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 9 lines
12-21 02:58:02.259 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@ce88ad9
12-21 02:58:02.961 11352 11352 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
12-21 02:58:02.963 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:02.969 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@5d3a281
12-21 02:58:02.972 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:02.972 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:02.973 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:02.973 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:02.974 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:02.974 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:02.975 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:02.975 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 29 lines
12-21 02:58:02.975 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:02.977 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@f0d5767
12-21 02:58:02.977 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:02.977 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:02.984 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@5d3a281
12-21 02:58:03.008 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 9 lines
12-21 02:58:03.008 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@5d3a281
12-21 02:58:03.792 11352 11352 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
12-21 02:58:03.794 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:03.800 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@9f4c62
12-21 02:58:03.802 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:03.802 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:03.803 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:03.803 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:03.804 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:03.804 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:03.805 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:03.805 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 29 lines
12-21 02:58:03.805 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:03.806 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@3295eb0
12-21 02:58:03.806 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:03.806 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:03.814 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@9f4c62
12-21 02:58:03.838 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 9 lines
12-21 02:58:03.838 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@9f4c62
12-21 02:58:05.325 11352 11352 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
12-21 02:58:05.327 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:05.333 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@9f6a55b
12-21 02:58:05.336 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:05.336 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:05.338 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:05.338 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:05.339 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:05.339 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:05.340 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:05.340 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 29 lines
12-21 02:58:05.340 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:05.342 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@314a6d1
12-21 02:58:05.342 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:05.342 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:05.351 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@9f6a55b
12-21 02:58:05.375 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 9 lines
12-21 02:58:05.375 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@9f6a55b
12-21 02:58:09.130 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@dc2a26a
12-21 02:58:10.360 11352 11448 D TwiFucker: Removed promoted timeline tweet
12-21 02:58:10.449 11352 11352 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
12-21 02:58:10.452 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:10.463 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@8f623eb
12-21 02:58:10.467 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:10.467 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:10.470 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:10.470 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:10.473 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:10.473 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:10.473 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:10.474 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 29 lines
12-21 02:58:10.474 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:10.476 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@3222fe1
12-21 02:58:10.477 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:10.477 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:10.492 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@8f623eb
12-21 02:58:10.531 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 9 lines
12-21 02:58:10.531 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@8f623eb
12-21 02:58:10.535 11352 11352 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
12-21 02:58:10.536 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:10.540 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@1f16330
12-21 02:58:10.542 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:10.542 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:10.543 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:10.543 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:10.544 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:10.544 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:10.544 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:10.545 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 29 lines
12-21 02:58:10.545 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:10.546 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@9e8a52e
12-21 02:58:10.546 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:10.546 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:10.554 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@1f16330
12-21 02:58:10.554 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 6 lines
12-21 02:58:10.554 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@1f16330
12-21 02:58:10.569 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@9f84ec6
12-21 02:58:10.585 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@1f16330
12-21 02:58:10.585 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 1 line
12-21 02:58:10.585 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@1f16330
12-21 02:58:10.586 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@9f84ec6
12-21 02:58:10.586 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@9f84ec6
12-21 02:58:10.587 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@647e1c4
12-21 02:58:10.587 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 4 lines
12-21 02:58:10.587 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@647e1c4
12-21 02:58:10.595 11352 11433 W LOTTIE  : Animation contains merge paths. Merge paths are only supported on KitKat+ and must be manually enabled by calling enableMergePathsForKitKatAndAbove().
12-21 02:58:11.530 11352 11352 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
12-21 02:58:11.532 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:11.537 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@c10dbdb
12-21 02:58:11.540 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:11.540 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:11.541 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:11.541 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:11.542 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:11.542 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:11.542 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:11.542 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 29 lines
12-21 02:58:11.542 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:11.543 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@19e0151
12-21 02:58:11.544 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:11.544 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:11.551 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@c10dbdb
12-21 02:58:11.588 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 9 lines
12-21 02:58:11.588 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@c10dbdb
12-21 02:58:11.589 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@14962bb
12-21 02:58:11.589 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.StateListDrawable@14962bb
12-21 02:58:12.312 11352 11352 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
12-21 02:58:12.314 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:12.319 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@aa5f014
12-21 02:58:12.322 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:12.322 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:12.323 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:12.323 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:12.324 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:12.324 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:12.324 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:12.325 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 29 lines
12-21 02:58:12.325 11352 11352 W ConstraintHelper: Parent not a ConstraintLayout
12-21 02:58:12.326 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@8ff96b2
12-21 02:58:12.326 11352 11352 E ConstraintLayout: layout_constraintWidth_default="wrap" is deprecated.
12-21 02:58:12.326 11352 11352 E ConstraintLayout: Use layout_width="WRAP_CONTENT" and layout_constrainedWidth="true" instead.
12-21 02:58:12.333 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@aa5f014
12-21 02:58:12.357 11352 11352 I chatty  : uid=10177(com.twitter.android) identical 9 lines
12-21 02:58:12.357 11352 11352 W unknown:WrappingUtils: Don't know how to round that drawable: android.graphics.drawable.RippleDrawable@aa5f014
12-21 02:58:20.975 11352 11411 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=4337c65a-681f-4c8f-8206-e8a7439566c3, tags={ com.twitter.analytics.service.core.workmanager.AnalyticsFlushWorker } ]
12-21 02:58:22.130 11352 11404 W SQLiteLog: (28) double-quoted string literal: "0"
12-21 02:59:12.645 11352 11352 W IInputConnectionWrapper: requestCursorAnchorInfo on inactive InputConnection
12-21 02:59:12.647 11352 11352 W IInputConnectionWrapper: requestCursorAnchorInfo on inactive InputConnection
```

**Comment by defkev** [comment]
Still present in Twitter 9.70.0-release.0 w/ TwiFucker 1.8

For whats it worth enabling **Use legacy hook** seems to fix it.

**Comment by defkev** [comment]
Can confirm this is fixed in Twitter 9.72.0-release.0 w/ TwiFucker 1.9 👍
